ckanext/data_qld/reporting_plugin.py

```diff
@@ -0,0 +1,69 @@
+import ckan.plugins as plugins
+import ckanext.data_qld.reporting.logic.action.get as get
+import auth_functions as auth
+
+from ckanext.data_qld.reporting.helpers import helpers
+
+
+class ReportingPlugin(plugins.SingletonPlugin):
+    plugins.implements(plugins.IConfigurer)
+    plugins.implements(plugins.IActions, inherit=True)
+    plugins.implements(plugins.IRoutes, inherit=True)
+    plugins.implements(plugins.ITemplateHelpers, inherit=True)
+    plugins.implements(plugins.IAuthFunctions)
+
+    # IConfigurer
+    def update_config(self, config_):
+        plugins.toolkit.add_resource('reporting/fanstatic', 'data_qld_reporting')
+
+    # IActions
+    def get_actions(self):
+        actions = {
+            'organisation_followers': get.organisation_followers,
+            'dataset_followers': get.dataset_followers,
+            'dataset_comments': get.dataset_comments,
+            'dataset_comment_followers': get.dataset_comment_followers,
+            'datasets_min_one_comment_follower': get.datasets_min_one_comment_follower,
+            'datarequests_min_one_comment_follower': get.datarequests_min_one_comment_follower,
+            'datarequests': get.datarequests,
+            'datarequest_comments': get.datarequest_comments,
+            'dataset_comments_no_replies_after_x_days': get.dataset_comments_no_replies_after_x_days,
+            'datarequests_no_replies_after_x_days': get.datarequests_no_replies_after_x_days,
+            'datarequests_for_circumstance': get.datarequests_for_circumstance,
+            'open_datarequests_no_comments_after_x_days': get.open_datarequests_no_comments_after_x_days,
+            'datarequests_open_after_x_days': get.datarequests_open_after_x_days,
+            'comments_no_replies_after_x_days': get.comments_no_replies_after_x_days,
+        }
+        return actions
+
+    # IRoutes
+
+    def before_map(self, map):
```

Comment:
@duttonw this plugin is not using the `ckanext-report` extension. This is a separate plugin. It sits under dashboard because that's where the functionality is accessed from, i.e. the new tab on the dashboard menu "My reports".